src/stores/room-list/algorithms/list-ordering/ImportanceAlgorithm.ts

```diff
@@ -136,6 +136,9 @@ export class ImportanceAlgorithm extends OrderingAlgorithm {
         } else {
             throw new Error(`Unhandled splice: ${cause}`);
         }
+
+        // changes have been made if we made it here, so say so
+        return true;
```

Comment:
Why wasn't that caught as a type error...?

Reply:
Because async :(